setup.py

```diff
@@ -198,7 +198,7 @@ def get_sphinx_theme_version() -> str:
     'watchtower~=0.7.3',
 ]
 apache_beam = [
-    'apache-beam[gcp]',
+    'apache-beam>=2.20.0',
```

Comment:
gcp extra is required by Dataflow operators.

Reply:
Good point. I added it to additional-extras for "apache.beam" in google provider. As the result this is what google provider extras will look like:

```
        extras_require={
            'amazon': ['apache-airflow-providers-amazon'],
            'apache.beam': ['apache-airflow-providers-apache-beam', 'apache-beam[gcp]'],
            'apache.cassandra': ['apache-airflow-providers-apache-cassandra'],
            'cncf.kubernetes': ['apache-airflow-providers-cncf-kubernetes'],
            'facebook': ['apache-airflow-providers-facebook'],
            'microsoft.azure': ['apache-airflow-providers-microsoft-azure'],
            'microsoft.mssql': ['apache-airflow-providers-microsoft-mssql'],
            'mysql': ['apache-airflow-providers-mysql'],
            'oracle': ['apache-airflow-providers-oracle'],
            'postgres': ['apache-airflow-providers-postgres'],
            'presto': ['apache-airflow-providers-presto'],
            'salesforce': ['apache-airflow-providers-salesforce'],
            'sftp': ['apache-airflow-providers-sftp'],
            'ssh': ['apache-airflow-providers-ssh'],
            'trino': ['apache-airflow-providers-trino'],
        },
```

Anyone willing to use DataFlow operators will have to use `apache-airflow-providers-google[apache.beam]`. Those extras are nicely documented in the README,